aws/flatcar-linux/kubernetes/ami.tf

```diff
@@ -1,13 +1,15 @@
 locals {
-  # Pick a CoreOS Container Linux derivative
-  ami_id = local.flavor == "flatcar" ? data.aws_ami.flatcar.image_id : data.aws_ami.coreos.image_id
+  # Pick a Container Linux derivative
+  ami_id = "${element(concat(data.aws_ami.flatcar.*.image_id, data.aws_ami.coreos.*.image_id, list("")), 0)}"
```

Comment:
Why to use string interpolation is that is not needed in Terraform 0.12? :sob:

Reply:
Oops :man_facepalming: Nice catch @mauriciovasquezbernal!

Reply:
I am not aware of the change in Terraform 0.12. How would you write this line instead?

Reply:
You just need to remove quotes and interpolation Syntax and you just leave pure expression.